src/core/util/util.spec.js

```diff
@@ -223,10 +223,33 @@ describe('util', function() {
         // Restore the scrolling.
         enableScrolling();
         window.scrollTo(0, 0);
-        document.body.removeChild(element[0]);
+
+        element.remove();
       }));
     });
 
+    describe('getViewportTop', function() {
```

Comment:
The viewport is not the `md-content`. `Window` is always the viewport, and `getBoundingClientRect()` is always in relative to the viewport.